fabric-object-builder-api-v1/src/testmod/java/net/fabricmc/fabric/test/object/builder/CriterionRegistryTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.object.builder;
+
+import com.google.gson.JsonObject;
+
+import net.minecraft.advancement.criterion.ImpossibleCriterion;
+import net.minecraft.predicate.entity.AdvancementEntityPredicateDeserializer;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.object.builder.v1.advancement.FabricCriterionRegistry;
+
+public final class CriterionRegistryTest {
+	public static void init() {
```

Comment:
why not just implement `ModInitializer` here for the tests rather than use method references. The `onInitialize` method is more obvious.

Reply:
This one static method imo is more clean and efficient than actually implementing one interface and have concern about what to do in constructors.

Reply:
I'd use ModInitializer since that's sorta what it's there for but doesn't really matter

Reply:
Fabric dimensions use static init as well

Reply:
I would say use `ModInitializer`, the existing test mods have that, and I think its the more common way to do it.